monitor/monitoring-dam-generator-core/src/main/java/eu/seaclouds/monitor/monitoringDamGenerator/adpParsing/Host.java

```diff
@@ -0,0 +1,47 @@
+package eu.seaclouds.monitor.monitoringDamGenerator.adpParsing;
+
+import it.polimi.tower4clouds.rules.MonitoringRules;
+
+import java.util.Map;
+
+public class Host {
```

Comment:
can I consider this https://docs.oracle.com/javase/tutorial/essential/concurrency/imstrat.html as a valid guide line for building immutable objects? SHould this apply also to the Module class?